### ISSUE TITLE
MongoDB addition causes provision failures on new installs

### ISSUE THREAD

**Issue by srrobinson** [laravel/homestead#426]
## Versions
  - Vagrant: 1.9.1
  - Provider: Virtualbox 5.1.12
  - Homestead: 4.0.3

## Host operating system
Windows 10 pro X64

## Homestead.yaml
```yaml
# Copy-paste your Homestead.yaml here

---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: C:/dev/
      to: /home/vagrant/Code

sites:
    - map: homestead.app
      to: /home/vagrant/Code/Laravel/public

databases:
    - homestead

# blackfire:
#     - id: foo
#       token: bar
#       client-id: foo
#       client-token: bar

# ports:
#     - send: 50000
#       to: 5000
#     - send: 7777
#       to: 777
#       protocol: udp
``` 

## Vagrant up output
https://gist.github.com/srrobinson/370333793ad0f78ce91874851e480ee0

## Expected behavior
Provision should not have attempted a mongodb deploy by default or a mongo parameter should have been added to Homestead.yaml to set true/false and be obeyed (manually adding mongodb: false parameter doesn't alleviate issue)

## Actual behavior
Provision failed during db creation and hence is incomplete. 

## Steps to reproduce
1. Follow steps in Laravel docs to create homestead instance and configure Homestead.yaml file
2. Issue 'vagrant up'
3. Hit Mongodb failure. 

## Workaround
Clear out all mongodb code from scripts/homestead.rb, provisioning then completes fine. 

## References
N/A

**Comment by svpernova09** [comment]
You're using `master`, not `4.0.3`. Please make sure you're reporting the right versions.


Thanks for the report, this has been fixed via #427

**Comment by srrobinson** [comment]
Issue resolution confirmed, thank you.
